initial-configuration/mysql-docker/docker-compose.yml

```
@@ -15,8 +13,9 @@ services:
       test: [ "CMD", "mysqladmin" ,"ping", "-uroot", "-p$PICSURE_DB_ROOT_PASS" ]
       timeout: 20s
       retries: 10
+      start_period: 15s
     volumes:
-      - $DOCKER_CONFIG_DIR/picsure-db:/var/lib/mysql
+      - $DOCKER_CONFIG_DIR/picsure-db:/var/lib/mysqld
```

```suggestion
      - $DOCKER_CONFIG_DIR/picsure-db:/var/lib/mysql
```

This needs to be changed back and I need to determine why the mysql mount doesn't work.